### ISSUE TITLE
Investigate spam filtering on mailbox.org

### ISSUE THREAD

**Issue by JasonBarnabe** [JasonBarnabe/greasyfork#1117]
Example headers for a confirmation email:

```
X-Spam-Flag: YES
X-Spam-Score: 16.011
X-Spam-Level: ****************
X-Spam-Status: Yes, score=16.011 tagged_above=2 required=6.31
	tests=[DKIM_SIGNED=0.1, DKIM_VALID=-0.1, DKIM_VALID_AU=-0.1,
	DKIM_VALID_EF=-0.1, HS_RSPAMD_20_21=9, HTML_MESSAGE=0.001,
	HTML_MIME_NO_HTML_TAG=0.377, MIME_HTML_ONLY=0.1, RCVD_IN_SBL_CSS=1.8,
	RCVD_IN_XBL=0.375, RCVD_IN_ZEN_LASTEXTERNAL=1, RDNS_NONE=0.793,
	SPF_HELO_NONE=0.001, SPF_PASS=-0.001, VFY_ACCT_NORDNS=2.765]
	autolearn=no autolearn_force=no
```

**Comment by JasonBarnabe** [comment]
HTML_MIME_NO_HTML_TAG should be fixed by 40bade0693c56a76d8ddf79fd689a6c7c47a47e4.

VFY_ACCT_NORDNS may be due to the server identifying itself as localhost.localdomain instead of mail.greasyfork.org. I think I may have fixed this.

**Comment by JasonBarnabe** [comment]
```
Nov 24 21:09:33 mail postfix/smtp[4087117]: 76040A28A9: host mx2.mailbox.org[80.241.60.215] said: 450 4.2.0 <greasyforktest@mailbox.org>: Recipient address rejected: Greylisted, see http://postgrey.schweikert.ch/help/mailbox.org.html (in reply to RCPT TO command)
Nov 24 21:09:35 mail postfix/smtp[4087117]: 76040A28A9: to=<greasyforktest@mailbox.org>, relay=mx2.mailbox.org[2001:67c:2050:104:0:2:25:1]:25, delay=6.2, delays=0.1/0.01/5.5/0.55, dsn=4.2.0, status=deferred (host mx2.mailbox.org[2001:67c:2050:104:0:2:25:1] said: 450 4.2.0 <greasyforktest@mailbox.org>: Recipient address rejected: Greylisted, see http://postgrey.schweikert.ch/help/mailbox.org.html (in reply to RCPT TO command))
```

**Comment by JasonBarnabe** [comment]
This seems to be normal operation of mailbox.org - first message is deferred for 10 minutes and then goes through successfully. http://projects.puremagic.com/greylisting/

So mailbox.org seems to be receiving emails from Greasy Fork - presumably this has been fixed by the changes above or stuff in #1199.
